github/event.go

```diff
@@ -129,8 +129,7 @@ func (e *Event) ParsePayload() (payload interface{}, err error) {
 // Deprecated: Use ParsePayload instead, which returns an error
 // rather than panics if JSON unmarshaling raw payload fails.
 func (e *Event) Payload() (payload interface{}) {
-	var err error
-	payload, err = e.ParsePayload()
+	payload, err := e.ParsePayload()
```

Comment:
If you're going to make this change, then you should also change line 131 and remove the named return variable because you just declared a new `payload` and they are no longer the same thing.